mlir/test/Dialect/Transform/Library/lower-to-llvm.mlir

```diff
@@ -0,0 +1,48 @@
+// RUN: mlir-opt %s
```

Comment:
This test does not do anything?

Reply:
seems I made a mistake and erased the relevant file when renaming indeed, thanks for noticing!
I'll add the missing test file tomorrow and report back here.

Reply:
#69190 already adds tests for all I could think of testing but feel free to add more if there's something I missed.